src/AutoMapper/QueryableExtensions/ProjectionBuilder.cs

```diff
@@ -64,7 +64,10 @@ public QueryExpressions CreateProjection(in ProjectionRequest request, LetProper
     QueryExpressions CreateProjection(in ProjectionRequest request, LetPropertyMaps letPropertyMaps, TypeMap typeMap, TypeMap[] polymorphicMaps)
     {
         var instanceParameter = Parameter(request.SourceType, "dto" + request.SourceType.Name);
-        var projection = CreateProjectionCore(request, instanceParameter, typeMap, letPropertyMaps);
+        var projection = !typeMap.DestinationType.IsAbstract
```

Comment:
A better fix is to simply skip abstract types when computing polymorphic maps.

Reply:
I disagree with that, or perhaps I don't see how to do it this way at this level.
Because the abstract type mapping don't come from the polymorphic map but from the EnumerableProjectionMapper.Project method.
This is because it is this abstract type that is used for the collection property in the entity.

Reply:
I guess both make sense, projecting to an abstract type when there are included maps and also including an abstract map, presumably for configuration reuse.